Sources/SwiftParser/generated/Parser+Entry.swift

```diff
@@ -146,11 +146,15 @@ fileprivate extension Parser {
       return into
     }
 
-    let remainingTokens = self.consumeRemainingTokens()
+    var remainingTokens = self.consumeRemainingTokens()
     if remainingTokens.isEmpty {
       return into
     }
 
+    if let existingUnexpected = layout.children[layout.children.count - 1] {
+      assert(existingUnexpected.is(RawUnexpectedNodesSyntax.self))
+      remainingTokens.insert(contentsOf: existingUnexpected.as(RawUnexpectedNodesSyntax.self)!.elements, at: 0)
```

Comment:
Could just use `.cast` here. FWIW I'd prefer something like:
```
var remainingTokens = self.consumeRemainingTokens()
if remainingTokens.isEmpty {
  return into
}

let existingUnexpected: [...]
if let unexpectedNode = layout.children[layout.children.count - 1] {
  assert(...) // we could probably move this assert into the cast so that we get a nice message rather than the force unwrap one
  existingUnexpected = unexpectedNode.cast(RawUnexpectedNodesSyntax.self).elements
} else {
  existingUnexpected = []
}
let unexpected = RawUnexpectedNodesSyntax(elements: existingUnexpected + remainingTokens, arena: self.arena)
```

But either is fine.

Reply:
> Could just use .cast here

Nope. `cast` does not exist on `RawSyntax`. I’m not sure if this use case is worth adding `case`. `as(…)!` works equally fine.

Reply:
The split still gets me :P!